### PR TITLE
docs(source-wordpress-experimental): update to wordpress sourcing guide

### PR DESCRIPTION
**Pull request by moonmeister**
## Description

Updates the sourcing guide to reflect the new WordPress source plugin.

### CODE REVIEW

**Comment by moonmeister** [comment]
This needs to be merged 7/6 for the Beta WP integration launch.

File: docs/docs/headless-cms.md [hunk 1]
```diff
@@ -21,33 +21,33 @@ The guides in this section will walk through the process of setting up content s
...
+| --------------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------------------ |
+| [Contentful](https://www.contentful.com/)     | [guide](/docs/sourcing-from-contentful/)                                         | [docs](/packages/gatsby-source-contentful)             | [starter](/starters/contentful/starter-gatsby-blog/)               |
+| [NetlifyCMS](https://www.netlifycms.org/)     | [guide](/docs/sourcing-from-netlify-cms/)                                        | [docs](/packages/gatsby-plugin-netlify-cms)            | [starter](/starters/netlify-templates/gatsby-starter-netlify-cms/) |
+| [WordPress](https://www.wordpress.com/)       | [guide](/docs/sourcing-from-wordpress/)                                          | [docs](/packages/gatsby-source-wordpress-experimental) |                                                                    |
```
Comment (TylerBarnes) [inline docs/docs/headless-cms.md:28]: The guide here looks like it's a 404. Maybe it should link here? https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/blob/master/docs/getting-started.md

Or maybe this link will exist once this pr is merged?

Comment (moonmeister) [reply docs/docs/headless-cms.md:28]: It should already exist cause this is an update, not a new guide

Comment (moonmeister) [reply docs/docs/headless-cms.md:28]: it works...https://www.gatsbyjs.org/docs/sourcing-from-wordpress/

Comment (TylerBarnes) [reply docs/docs/headless-cms.md:28]: ah, my bad, I went to the rich view and clicked the link. it's a relative link 🤦

**Comment by TylerBarnes** [comment]
Thanks again for this @moonmeister !

File: docs/docs/headless-cms.md (hunk 1 above)
Comment (moonmeister) [reply docs/docs/headless-cms.md:28]: Thanks, please merge, I can't on this repo.

Comment (TylerBarnes) [reply docs/docs/headless-cms.md:28]: It should merge once all the tests pass

**Pull request merged**